handbook/marketing/messaging.md

```diff
@@ -64,7 +64,7 @@ Sourcegraph is a company that creates software to help every organization build
 
 ## Press release boilerplate
 
-Sourcegraph empowers all developers with an elite set of solutions for universal code search, intelligence, and automation. Code intelligence can help bring the future sooner, and Sourcegraph’s mission is to provide tools that let developers from any organization create quality products by leveraging the best technology available. Sourcegraph is built for companies of all sizes, from startups that need a solid foundation for growth, to the world’s largest enterprises with complex security, scaling, and deployment needs. The company is headquartered in San Francisco, CA, and is backed by Redpoint Ventures and Goldcrest Capital. Follow Sourcegraph on Twitter at [@srcgraph](https://twitter.com/srcgraph) or visit [Sourcegraph.com](https://sourcegraph.com).
+Sourcegraph empowers all developers with universal code search. Sourcegraph’s mission is to provide tools that let developers from any organization create quality products by leveraging the best technology available. Sourcegraph is built for companies of all sizes, from startups that need a solid foundation for growth, to the world’s largest enterprises with complex security, scaling, and deployment needs. The company is headquartered in San Francisco, CA, and is backed by Redpoint Ventures and Goldcrest Capital. Follow Sourcegraph on Twitter at [@srcgraph](https://twitter.com/srcgraph) or visit [Sourcegraph.com](https://sourcegraph.com).
```

Comment:
Should we remove headquarterd in SF since we are remote? Or do we maintain this with our new address?